README.md

```diff
@@ -38,35 +38,65 @@ The bounty board will be a key mechanism for coordinating talent, tasks and capi
 
 For the MVP, we are focusing on the bare requirements for a Bounty Card to be created by a user via DEGEN and/or Frontend UI, with the following key fields:
 
-- **season**: In what season is this bounty being created for?
 - **Title**: Bounty Titles should be like headlines
 - **Description**: Provides space to flesh out the scope, deliverables and timeline for the bounty.
 - **Criteria**: When is a task considered "done" or "complete"?
-- **Reward**: Bounty creator indicates `currency` (i.e., $BANK) and `amount` (i.e., 10000) to be paid for completing the work.
-- **CreatedBy**: Bounty creator provides their `discordHandle` and `discordId` is automatically generated.
-- **CreatedAt**: The date in which the bounty is created.
-- **DueAt**: The deadline for completing the work.
+- **Reward**: Bounty creator indicates `amount` (i.e., 10000) and `currency` (i.e., $BANK) to be paid for completing the work.
+- **HashId**: Auto-generated ID for each bounty.
+- **CreatedBy / RequestedBy**: Bot automatically enters bounty creator's `discordHandle`.
+- **ClaimedBy**: Bot automatially notes `discordHandle` claiming the bounty.
+- **SubmittedBy**: Bot automatically notes `discordHandle` submitting the bounty.
 
-#### Bounty Card Status
+#### User Experience Flow
 
-TBD
+The User Experience flow with accompanying bounty status is as follows:
 
-#### Users
+1. **Open**: Bounty creator creates new bounty; status is now "Open" in Discord.
```

Comment:
I would change this to 

1. **Draft**: Bounty creator creates new bounty; status is now "Draft" and **not** shown in Discord. Bounty creator must publish the bounty before it is available to the public.